Source/monster.cpp

```diff
@@ -4368,7 +4369,7 @@ void ProcessMonsters()
 				raflag = MonsterTalk(monster);
 				break;
 			}
-			if (raflag) {
+			if (raflag || monster._mmode == MonsterMode::Stand) {

```

Comment:
Why do we check `Stand` here?
I ask cause gargoyles don't use stand when sitting on the ground and I'm unsure if we need to address this also here.

Reply:
To see if they are attempting to move but unable to do so. For gargoyles, they are prohibited from sleeping when loading a group i believe